static/app/components/replays/videoReplayer.tsx

```diff
@@ -24,7 +24,7 @@ interface VideoReplayerOptions {
   clipWindow?: ClipWindow;
 }
 
-interface VideoReplayerConfig {
+export interface VideoReplayerConfig {
```

Comment:
Should this be moved into a `types` file?

Reply:
i think it makes sense to keep it here - it's only accessed in the new wrapper player outside of this file. we don't have a types file in this folder yet so don't know if it would make sense to create a new one